implementation/src/main/java/io/smallrye/openapi/runtime/scanner/OpenApiAnnotationScanner.java

```diff
@@ -384,6 +458,19 @@ private void processJaxRsResourceClass(OpenAPIImpl openApi, ClassInfo resourceCl
         }
     }
 
+    void addScopes(String[] roles) {
```

Comment:
I think this could use a little bit of javadoc.

Reply:
@EricWittmann - added the method JavaDoc. Please let me know if there is anything else.